repository-data/webfiles/src/main/resources/site/freemarker/common/button.ftl

```diff
@@ -0,0 +1,35 @@
+<#ftl output_format="HTML">
+
+<#include "../include/imports.ftl">
+<#include "macro/metaTags.ftl">
+
+<#-- Add meta tags -->
+<@metaTags></@metaTags>
```

Comment:
I don't think the metaTags are needed on the catalog component.
Worth checking if they're already added to the DOM by an ftl higher up.

Reply:
This has been removed